src/clr-addons/readonly-combobox/readonly-combobox.directive.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2018-2024 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: 'clr-combobox-container[readOnly], clr-datalist-container[readOnly], clr-select-container[readOnly]',
+})
+export class ClrReadonlyCombobox {
+  constructor(private elRef: ElementRef, private renderer: Renderer2) {}
+
+  @Input() set readOnly(value: boolean) {
+    const inputElement = this.elRef.nativeElement.querySelector('input, select');
+    if (value) {
+      setTimeout(() => {
+        this.renderer.setAttribute(inputElement, 'disabled', 'true');
+        this.renderer.setAttribute(this.elRef.nativeElement, 'readonly', 'true');
+      });
+    } else {
+      this.renderer.removeAttribute(inputElement, 'disabled');
```

Comment:
Do we want to support disabled and readonly elements separately? if the element was disabled before using this directive, this else branch would remove the "default" disabled state

If you say that is out of scope, then it's fine